src/NUnitFramework/framework/Internal/PropertyBag.cs

```diff
@@ -49,6 +49,8 @@ public class PropertyBag : IPropertyBag
         /// <param name="value">The value</param>
         public void Add(string key, object value)
         {
+            Guard.ArgumentNotNull(value, "value");
```

Comment:
If CI hasn't passed by the time you see this, I'd really like to have `nameof(value)` here. I've seen the top line of an NUnit method throw an exception that had an old param name in it.

Reply:
Coincidentally I thought of doing it this way but I was dissuaded by rarely seeing it done in the rest of the codebase.  :(

Reply:
Nice 🙃 @ChrisMaddock did an initial sweep to get us onto `nameof` in https://github.com/nunit/nunit/pull/2623.